bundles/org.openhab.binding.melcloud/src/main/resources/OH-INF/thing/heatpumpDevice.xml

```diff
@@ -34,10 +34,10 @@
 				<label>Building ID</label>
 				<description>Building ID of the Heatpump device.</description>
 			</parameter>
-			<parameter name="pollingInterval" type="integer" required="true">
+			<parameter name="pollingInterval" type="integer" min="300" required="true">
 				<label>Polling Interval</label>
 				<description>Time interval how often poll data from MELCloud</description>
-				<default>60</default>
+				<default>360</default>
```

Comment:
Can you also update the readme with this new default?

Reply:
Ah yes... missed that one. Thanks, will update.